.circleci/config.yml

```diff
@@ -146,17 +146,15 @@ jobs:
     executor: docker-python
     steps:
       - checkout
-      - setup_remote_docker:
-          docker_layer_caching: true
+      - setup_remote_docker
       - run:
           name: Run tests
           command: make test
   build-and-test-v2:
     executor: docker-python
     steps:
       - checkout
-      - setup_remote_docker:
-          docker_layer_caching: true
+      - setup_remote_docker
```

Comment:
I suspect the layer caching increased the flakiness - didn't speed up the tests so may as well remove